### DISCUSSION TITLE
How to set the correct preset for my data

### DISCUSSION THREAD

**Discussion by pandahui01** [milaboratory/mixcr#1870] [Q&A]
This is my align.txt

Analysis time: 31.56s
Total sequencing reads: 9035215
Successfully aligned reads: 529014 (5.86%)
Coverage (percent of successfully aligned):
  CDR3: 431900 (81.64%)
  FR3_TO_FR4: 405582 (76.67%)
  CDR2_TO_FR4: 404692 (76.5%)
  FR2_TO_FR4: 394254 (74.53%)
  CDR1_TO_FR4: 393549 (74.39%)
  VDJRegion: 412315 (77.94%)
Alignment failed: no hits (not TCR/IG?): 6828336 (75.57%)
Alignment failed: absence of V hits: 5955 (0.07%)
Alignment failed: absence of J hits: 1671910 (18.5%)

I looked at the previous preset discussion and used 10x-bulk.yaml because my data came with umi, and did breaks to allow it to be assembled into a full-length sequence (PE150)

Can you help me? Thank you!

**Comment by mizraelson** [comment]
Hi, yes the 10x-bulk.yaml preset should work, but you will have to specify the pattern manually that describes you UMI structure.

**Reply by pandahui01** [reply]
The format of my.fq files is as follows:
	read1.fq: 25bp (barcode)+10bp(umi)
	read2.fq: 150bp (vdj)
	I'm facing an issue where I'm unsure how to accurately specify/define my multiple barcode sequences as a single cell for analysis. I've made an attempt using 10x-bulk.yaml, considering 35bp as umi, and up are the results I got.
        So I used 10x.yaml .I generated the whitelist myself, but get error
        can you help me to set the suitable preset for me?

**Reply by mizraelson** [reply]
I am not sure I understand. Is it a single cell data or not? In R1: 25bp (barcode)+10bp(umi) what is the 25bp barcode?

**Comment by pandahui01** [comment]
I am sorry for my late reply. It is stereo-seq data.&nbsp;the 25bp barcode&nbsp;is like a dnb code, One cell covers many dnbs.&nbsp;


熊喵
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2025年1月28日(星期二) 上午10:47
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [milaboratory/mixcr] How to set the correct preset for my data (Discussion #1870)














 
I am not sure I understand. Is it a single cell data or not? In R1: 25bp (barcode)+10bp(umi) what is the 25bp barcode?

**Comment by mizraelson** [comment]
So if I understand correctly, this is a spatial transcriptomics when per each 25bp barcode you might have several clonotypes. I would recommend trying the preset attached.
[stereo-seq.yaml.gz](https://github.com/user-attachments/files/18778203/stereo-seq.yaml.gz)
